libnetwork/drivers/overlay/ov_network.go

```diff
@@ -609,7 +609,16 @@ func (n *network) setupSubnetSandbox(s *subnet, brName, vxlanName string) error
 
 	err := createVxlan(vxlanName, s.vni, n.maxMTU())
 	if err != nil {
-		return err
+		if !strings.Contains(err.Error(), "file exists") {
+			return err
+		}
+		logrus.Warnf("The orphaned vxlan interface %s already exists in host network namespace, delete it", vxlanName)
+		if err := deleteInterface(vxlanName); err != nil {
```

Comment:
Curious; are these interfaces _guaranteed_ to be owned by us? IOW, could one exist that we didn't create? (and we'd be removing one that's created by something other than docker?

Reply:
In theory, there's no 100% guarantee. But the `vxlanName` has a special name scheme, it's very unlikely created by something other than docker.